ncm-systemd/src/main/perl/Systemd/Service/Unit.pm

```diff
@@ -442,9 +442,18 @@ sub configured_units
                 $self->error("Unitfile configuration failed, skipping the unit ", $self->unit_text($detail));
                 next;
             } elsif ($changed) {
-                $self->verbose("Going to issue conditional restart for $detail->{name} due to changed unitfile");
-                push(@tryrestart, $detail->{name});
-            };
+		$ufile->{config}->reset;
```

Comment:
i sort of missed this. seems to be related to #1453 but instead of configuring th eunit with 'Refuse', i disabled startstop.
but indeed, this is better detected automatically.

we can initiailse the unit_cache, we'll just have to do that again after th eunitfiles have been generated. not sure how slow it is.

however, you also need it where startstop is handled elsewhere. a unit filw 'Refuse' configured, should ignore startstop also in the Service code.

Reply:
@stdweird does this imply changes to this PR, or new issues to be opened?

Reply:
at the very least open an issue. i think it ow only takes into consideration self defined units, not existig ones.
the 2nd remark is more hypothetical (why would you flag a unit startstop if it has Refused configured)

open an issue, i can have a look at it after this is merged